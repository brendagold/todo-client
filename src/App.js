import "./App.css";
import Header from "./components/Header";
import Todos from "./components/Todos";
import { Provider } from "./context";

function App() {
  return (
    <Provider>
      <div className="App">
        <Header />
        
        <Todos />
      </div>
    </Provider>
  );
}

export default App;
