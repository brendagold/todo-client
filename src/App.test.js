import { render } from '@testing-library/react';
import App from './App';
import { shallow, mount } from 'enzyme';
import toJson from "enzyme-to-json";

// test('renders the app', () => {
//   const component = render(<App />);
//   expect(component).toMatchSnapshot();
// });

describe("rendering components", () => {
  it("renders App component without crashing", () => {
    shallow(<App />);
  });

});
