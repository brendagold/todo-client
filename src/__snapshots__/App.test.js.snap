// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the app 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="card bg-info text-center text-light rounded-0"
        >
          <h1
            class="display-4"
          >
            <i
              class="fas fa-clipboard-list mr-5"
            />
            <span
              class="text-dark mr-3 "
            >
              TODO
            </span>
             LIST
          </h1>
        </div>
        <form>
          <input
            class="form-control rounded-0"
            placeholder="write your todo here..."
            type="text"
            value=""
          />
          <button
            class="form-control rounded-0 btn-secondary"
            type="submit"
          >
            Add Todo
          </button>
        </form>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="card bg-info text-center text-light rounded-0"
      >
        <h1
          class="display-4"
        >
          <i
            class="fas fa-clipboard-list mr-5"
          />
          <span
            class="text-dark mr-3 "
          >
            TODO
          </span>
           LIST
        </h1>
      </div>
      <form>
        <input
          class="form-control rounded-0"
          placeholder="write your todo here..."
          type="text"
          value=""
        />
        <button
          class="form-control rounded-0 btn-secondary"
          type="submit"
        >
          Add Todo
        </button>
      </form>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
